lxd/resources/memory.go

```diff
@@ -144,6 +199,13 @@ func GetMemory() (*api.ResourcesMemory, error) {
 		return nil, errors.Wrap(err, "Failed to parse /proc/meminfo")
 	}
 
+	// Calculate the total memory from /sys/devices/system/memory.
+	// If successful, replace the previous value, retrieved from /proc/meminfo.
```

Comment:
@monstermunchkin would it be worth adding a comment here as to why we look up total memory instead of just using the /proc/meminfo value, for future reference.

Reply:
thanks